SETUP_SUBMITTER_CMF.md

```diff
@@ -0,0 +1,120 @@
+
+- Download the Epic Installer and install the latest version of Unreal (5.2 or higher is required)
+- NVidia Grid drivers - Follow windows instructions - https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/install-nvidia-driver.html#nvidia-GRID-driver
+
+The following steps: (“Install Build Tools”, "Environment Setup", "Deadline Software Installation", "Build the Plugin") should be followed whether you’re currently setting up your submitter machine or creating an instance/AMI to act as your worker node for Unreal jobs.
+
+## Install Build Tools
+
+The Unreal Submitter Plugin currently must be compiled locally.
+
+- Install Visual Studio 2022 or later using the Visual Studio Installer
+  - When installing, under “Workloads” select “Desktop development with C++”
+  - Under “Individual Components”, select a recent .NET Framework SDK (4.6.1 and 4.8.1 have been verified)
+
+## (Submitter only) Install Deadline Cloud Monitor
+
+Deadline Cloud Monitor is used to both manage your credentials for submitting jobs to Deadline Cloud as well as monitoring the status of your jobs.
+
+- Follow the instructions at https://docs.aws.amazon.com/deadline-cloud/latest/userguide/submitter.html#install-deadline-cloud-monitor
+- Sign in
+
+## Environment Setup
+
+- (If not already installed) Install a recent version of Python (3.12 has been verified)
+- Make sure your Environment Variables are set correctly. In System Environment Variables, your PATH should include:
+  - The path to your Python Installation (C:\Program Files\Python312 for example)
+  - The path to your Python Scripts folder (C:\Program Files\Python312\scripts for example)
+  - The path to your Unreal binaries (C:\Program Files\Epic Games\UE_5.4\Engine\Binaries\Win64)
+
+## Deadline Software Installation
+
+- (If this instance will be a worker agent) python -m pip install deadline-cloud-worker-agent
+- python -m pip install deadline-cloud-for-unreal-engine
+- clone or download deadline-cloud-for-unreal-engine
+
+## Build the Plugin
+
+Adjust the first two paths below based on where your installation of Unreal lives, and where you installed deadline-cloud-for-unreal-engine.
+
+From the Unreal Install Batchfiles Folder (Note the ‘package’ parameter can be any new directory, however you’ll want it to be called “UnrealDeadlineCloudService” later):
+
+- cd C:\Program Files\Epic Games\UE_5.4\Engine\Build\BatchFiles
+- runuat.bat BuildPlugin -plugin="C:\deadline\deadline-cloud-for-unreal-engine\src\unreal_plugin\UnrealDeadlineCloudService.uplugin" -package="C:\UnrealDeadlineCloudService"
+- Copy the “package” folder above to your Unreal installation’s Plugins folder (C:\Program Files\Epic Games\UE_5.4\Engine\Plugins\UnrealDeadlineCloudService for example)
+
+## (Submitter only) Submitter Installer
+
+Additional python libraries are installed by the submitter installer currently.
+
+- Download submitter installer from Deadline Cloud AWS Console’s Downloads Tab
```

Comment:
It can also be downloaded from Deadline Cloud Monitor app -> Workstation setup -> Downloads

Reply:
Will add a note!